allensdk/brain_observatory/behavior/behavior_session.py

```diff
@@ -176,22 +195,30 @@ def from_lims(cls, behavior_session_id: int,
         behavior_session_id = BehaviorSessionId(behavior_session_id)
         stimulus_file = BehaviorStimulusFile.from_lims(
             db=lims_db, behavior_session_id=behavior_session_id.value)
+
+        sync_file = None
```

Comment:
IMO this is easier to read as a one-liner 
```
sync_file = stimulus_timestamps._sync_file if stimulus_timestamps is not None else None
```

Alternatively it's easier to read as an if/else block